include/openspace/interaction/orbitalnavigator.h

```diff
@@ -142,6 +143,14 @@ class OrbitalNavigator : public properties::PropertyOwner {
     properties::FloatProperty _followAnchorNodeRotationDistance;
     properties::FloatProperty _minimumAllowedDistance;
 
+    properties::FloatProperty _velocitySensitivity;
+    properties::BoolProperty _flyTo;
+    properties::BoolProperty _overview;
+    //If 1st story selected, allow an additional camera update iteration before overview
+    bool _cameraInitializedBeforeFirstStoryOverview = false;
```

Comment:
I think we need to deal with this special case before merging it in - orbitalnavigator is such a core part of the system and should not need to care about the concept of a story

Reply:
Yea, I agree.  I quick workaround for that would be to set the camera position from the overview screen twice?  A bit hackish, but I think that would work

Reply:
@liuloppan  could you check if that works?
Removing this variable and the code that it executes and instead solving it on the GUI part?